deploy/example/vpa/uninstall-vpa.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set +o xtrace
+
+repo="https://github.com/kubernetes/autoscaler.git"
+echo "Uninstalling VPA ..."
+git clone $repo
+cd autoscaler/vertical-pod-autoscaler
```

Comment:
is there an easy way to install upstream autoscaler without cloning the whole repo? and it's better installing a stable version instead of master branch

Reply:
No, the doc only provides the installation to download the source code of VPA https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/docs/installation.md#install-command

Reply:
Note: the script currently reads environment variables: $REGISTRY and $TAG. Make sure you leave them unset unless you want to use a non-default version of VPA.